.github/workflows/ci.yml

```diff
@@ -97,6 +97,9 @@ jobs:
           # The main Error Prone run
           $MAVEN ${MAVEN_TEST} -T C1 clean test-compile -P gib,errorprone-compiler \
```

Comment:
Change `test-compile` to `install` and don't add another command. We can use `install` for consistency but `package` would be fine too since nothing will consume these artifacts.